Process "/home/pmaupin/projects/opensource/rst2pdf-0.93/bin/rst2pdf" started on Sun Jul  5 17:16:35 2015

        /home/pmaupin/projects/opensource/rst2pdf-0.93/bin/rst2pdf
        --date-invariant -v test_issue_419.txt -s test_issue_419.style
        -o
        /home/pmaupin/projects/opensource/rst2pdf-0.93/rst2pdf/tests/output/test_issue_419.pdf


* [INFO] createpdf.py:1572 Importing extension module 'vectorpdf'
* [INFO] languages.py:39 Language 'en_US' not supported by Docutils, using 'en' instead.
* [INFO] styles.py:82 Using stylesheets: test_issue_419.style
* [INFO] styles.py:110 FontPath:['/home/pmaupin/projects/opensource/rst2pdf-0.93/rst2pdf/fonts', '.']
* [INFO] styles.py:111 StylePath:['/home/pmaupin/.rst2pdf/styles', '/home/pmaupin/projects/opensource/rst2pdf-0.93/rst2pdf/styles', '.']
* [INFO] languages.py:39 Language 'en_US' not supported by Docutils, using 'en' instead.
* [INFO] basenodehandler.py:215 Unknown class admonition-test, ignoring. [near line UNKNOWN in file test_issue_419.txt]
* [INFO] createpdf.py:651 Starting build
* [INFO] createpdf.py:1112 Page 1 [1]

Program /home/pmaupin/projects/opensource/rst2pdf-0.93/bin/rst2pdf exit code: PASS (0)   elapsed time: 00.3 seconds

Validity of file test_issue_419.pdf checksum 'ba9909d3bf5807ab3826283d7b2d1331' is bad.
